src/Controller/HelperController.php

```diff
@@ -428,7 +428,8 @@ public function retrieveAutocompleteItemsAction(Request $request)
                     ));
                 }
 
-                $datagrid->setValue($property, null, $searchText);
+                $filter = $datagrid->getFilter($property);
+                $datagrid->setValue($filter->getFormName(), null, $searchText);
```

Comment:
No need to introduce a new variable for this IMO:
```php
<?php
$datagrid->setValue(
    $datagrid->getFilter($property)->getFormName(),
    null,
    $searchText
);
```

Reply:
@greg0ire sorry, I don't get you.

Without adding new variable to datagrid - filter would be ignored (current behavior)

Reply:
done